agent/tcs/handler/handler.go

```diff
@@ -35,8 +36,8 @@ const (
 	defaultPublishMetricsInterval = 20 * time.Second
 
 	// The maximum time to wait between heartbeats without disconnecting
-	defaultHeartbeatTimeout            = 5 * time.Minute
-	defaultHeartbeatJitter             = 3 * time.Minute
+	defaultHeartbeatTimeout            = 1 * time.Minute
```

Comment:
Why did we change these?

Reply:
I assume this is just meant to mimic the work that @richardpen did for the ACS client.

Reply:
@samuelkarp @jhaynes is correct. This is reflecting the change that @richardpen did.

Reply:
Got it.